cdc/cdc/sink/tikv.go

```diff
@@ -288,7 +296,11 @@ func (b *tikvBatcher) Append(entry *model.RawKVEntry) {
 		b.now = b.getNow()
 	}
 
-	opType, key, value, ttl := extractEntry(entry, b.now)
+	opType, key, value, ttl, err := extractEntry(entry, b.now)
+	if err != nil {
+		log.Warn("ignore this entry because of failed to extract entry", zap.Any("event", entry), zap.Error(err))
+		return
```

Comment:
Should raise error other than ignore silently.

Reply:
Emmm... Seems that the only solution to such illegal entries is ignored. Then suggest to add some metrics for this illegal entries, to inform users that what had happened.

Reply:
Added, PTAL~